compile.js

```diff
@@ -136,41 +137,41 @@ module.exports = function (schema, extraEncodings, inlineEnc) {
 
   visit(schema, '')
 
-  var compileEnum = function (e) {
-    var conditions = Object.keys(e.values)
+  const compileEnum = function (e) {
+    let conditions = Object.keys(e.values)
       .map(function (k) {
         return 'val !== ' + parseInt(e.values[k].value, 10)
       })
       .join(' && ')
 
     if (!conditions) conditions = 'true'
 
-    var encode = genfun()
-      ('function encode (val, buf, offset) {')
-        ('if (%s) throw new Error("Invalid enum value: "+val)', conditions)
-        ('varint.encode(val, buf, offset)')
-        ('encode.bytes = varint.encode.bytes')
-        ('return buf')
-      ('}')
+    const encode = genfun()
+    ('function encode (val, buf, offset) {')
+    ('if (%s) throw new Error("Invalid enum value: "+val)', conditions)
+    ('varint.encode(val, buf, offset)')
+    ('encode.bytes = varint.encode.bytes')
+    ('return buf')
+    ('}')
       .toFunction({
         varint: varint
       })
 
-    var decode = genfun()
-      ('function decode (buf, offset) {')
-        ('var val = varint.decode(buf, offset)')
-        ('if (%s) throw new Error("Invalid enum value: "+val)', conditions)
-        ('decode.bytes = varint.decode.bytes')
-        ('return val')
-      ('}')
+    const decode = genfun()
+    ('function decode (buf, offset) {')
+    ('var val = varint.decode(buf, offset)')
+    ('if (%s) throw new Error("Invalid enum value: "+val)', conditions)
+    ('decode.bytes = varint.decode.bytes')
+    ('return val')
+    ('}')
```

Comment:
indention here is intentional and standard limitations around that are disabled in the beginning of the file